src/rp2_common/pico_stdlib/stdlib.c

```diff
@@ -39,11 +39,13 @@ void set_sys_clock_48mhz() {
 // When the System clock PLL setup is changed from the initial setting made
 // during system initialisation, the code automatically switches the peripheral
 // clock to the 48MHz USB clock to ensure continuity of peripheral operation.
-// The Pico config option below allows a user to change this default behaviour if
-// required by defining the Pico config option below as 0 in their board file.
-// PICO_CONFIG: PICO_CLOCK_AJDUST_PERI_CLOCK_WITH_SYS_CLOCK, When the SYS clock PLL is changed also switch the peripheral clock to the USB clock, type=bool, default=1, advanced=true, group=hardware_clocks
+// The Pico config option below allows a user to change this default behaviour,
+// if required, by defining the Pico config option below as 1 in their board file.
+// If this option is selected, the peripheral clock will remain connected to the
+// system PLL clock and so will change frequency with that.
+// PICO_CONFIG: PICO_CLOCK_AJDUST_PERI_CLOCK_WITH_SYS_CLOCK, When the SYS clock PLL is changed also switch the peripheral clock to the USB clock, type=bool, default=0, advanced=true, group=hardware_clocks
```

Comment:
doc is now wrong

Reply:
It would appear so as rendered by github but it is not in the copy of file I see here even after:
`rm ./src/rp2_common/pico_stdlib/stdlib.c && git checkout -f ./src/rp2_common/pico_stdlib/stdlib.c`
Or in my `gitk` log.